### DISCUSSION TITLE
Event names not appearing

### DISCUSSION THREAD

**Discussion by pbkalra** [derekantrican/GAS-ICS-Sync#338] [Q&A]
Hello, thank you to those who created this script and provided such clear documentation!

I made a copy and installed the script a little while back and after that I was able to see my Outlook events in my Google calendar like I wanted to.  However, the events show up in Google calendar as "no title," although if I look at the details for the event, the title shows up in the notes field.  Does anyone know how I can fix this?  Thanks in advance.

**Comment by jonas0b1011001** [comment]
try setting this to true:
https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Code.gs#L42

**Reply by ValentinHerrmann** [reply]
I have the same problem. This did not solve it for me.  Does anyone have any other ideas? This is my config:

var howFrequent = 5; 
var onlyFutureEvents = false;
var addEventsToCalendar = true;
var modifyExistingEvents = true; 
var removeEventsFromCalendar = true;
var addAlerts = "no";
var addOrganizerToTitle = false;
var descriptionAsTitles = true;
var addCalToTitle = true;
var addAttendees = false;
var defaultAllDayReminder = -1;
                                      
var overrideVisibility = "";
var addTasks = false;

var emailSummary = false;
var email = "";

**Comment by ewkohler** [comment]
> Hello, thank you to those who created this script and provided such clear documentation!
> 
> I made a copy and installed the script a little while back and after that I was able to see my Outlook events in my Google calendar like I wanted to. However, the events show up in Google calendar as "no title," although if I look at the details for the event, the title shows up in the notes field. Does anyone know how I can fix this? Thanks in advance.

Try going to Outlook > Shared calendar options > select your calendar > Allow show all details
